Bots/DotNet/Consumers/CodeFirst/WaterfallHostBot/TokenExchangeSkillHandler.cs

```diff
@@ -29,39 +29,37 @@ public class TokenExchangeSkillHandler : SkillHandler
 
```

Comment:
In this class, there are some usings that are no longer needed. Also, the property WaterfallSkillBot can be deleted.